docs/traces.md

```diff
@@ -40,10 +40,10 @@ timestamp                                           | `start_time_unix_nano` fix
 `otel.library.name` tag                             | `InstrumentationLibrary.name` string        | `tags["otel.library.name"]`                                                      | `tags["otel.library.name"]`
 `otel.library.version` tag                          | `InstrumentationLibrary.version` string     | `tags["otel.library.version"]`                                                   | `tags["otel.library.version"]`
 .                                                   | `resource` Resource                         | `process` Process
-(free-form fields)\*                                | `Resource.attributes` repeated KeyValue.    | `process.tags` repeated KeyValue
+`otel.span.attributes` field string (json)          | `Resource.attributes` repeated KeyValue.    | `process.tags` repeated KeyValue
 .                                                   | `Resource.attributes["service.name"]`       | `process.service_name` string
 `otel.resource.dropped_attributes_count` field uint | `Resource.dropped_attributes_count` uint32
-(free-form fields)\*                                | `attributes` repeated KeyValue              | `tags` repeated KeyValue                                                         | `tags` map<string, string>
```

Comment:
@jacobmarble I'm pretty sure I misunderstood you here. I dumped both `Resource.attributes` and `attributes` into the same field, `otel.*.attributes`. Should they be split? Can there be multiple types of "resource" for a span? Let's walk through this later today.